api/src/proxy/proxy.service.spec.ts

```diff
@@ -29,12 +29,25 @@ describe('ProxyService', () => {
     const proxyData = {
       protocol: 'http',
       host: '127.0.0.1',
-      port: 80
+      port: 80,
+      location: 'kr'
     }
     const proxyObj = await proxy.create(proxyData)
     expect(proxyObj.protocol).toBe(proxyData.protocol)
     expect(proxyObj.host).toBe(proxyData.host)
     expect(proxyObj.port).toBe(proxyData.port)
+    expect(proxyObj.location).toBe(proxyData.location)
+
+    const proxyDataWithoutLocation = {
+      protocol: 'http',
+      host: '127.0.0.2',
+      port: 80
+    }
+
+    const proxyObjWithoutLocation = await proxy.create(proxyDataWithoutLocation)
+    expect(proxyObjWithoutLocation.protocol).toBe(proxyDataWithoutLocation.protocol)
+    expect(proxyObjWithoutLocation.host).toBe(proxyDataWithoutLocation.host)
+    expect(proxyObjWithoutLocation.port).toBe(proxyDataWithoutLocation.port)
```

Comment:
Shouldn't we test also on the value of not set `location` column?

Reply:
Do you mean I should make a case where `location` value is an empty string?

Reply:
On line 39 you check whether `location` value has been correctly sent, however, in the test below, you don't check for `location` value. We could check whether the value is `NULL`. Does it make sense?

Reply:
Now I understood, thank you

Reply:
updated based on the feedback :)